welcome/index.adoc

```diff
@@ -123,6 +123,8 @@ Use the
 xref:../applications/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[*Topology* view]
 to see your applications, monitor status, connect and group components, and modify your code base.
 
+- **xref:../applications/connecting_applications_to_services/understanding-service-binding-operator.adoc#understanding-service-binding-operator[Use the Service Binding Operator]**: The Service Binding Operator enables application developers to easily bind workloads together with Operator-managed backing services by automatically collecting and sharing binding data with the workloads. It enriches the development lifecycle with a consistent and declarative service binding method that eliminates discrepancies in cluster environments.
```

Comment:
I'm not sure why it's `Operator-managed` the hyphen seems unnecessary. Is this standard? I don't see it elsewhere in the documentation, so my thoughts is to remove it. So it would simply be `Operator managed.` Again, you know the technology more than I, so keep the hyphen if it's standard for Operator documentation :)

Reply:
@jc-berger Thanks for the suggestion, I am retaining the hyphen.